src/rendergraph/common/rendergraph/material/rgbamaterial.cpp

```diff
@@ -0,0 +1,34 @@
+#include "rgbamaterial.h"
+
+#include <QVector2D>
+
+#include "rendergraph/materialshader.h"
+#include "rendergraph/materialtype.h"
+#include "rendergraph/uniformset.h"
+
+using namespace rendergraph;
+
+RGBAMaterial::RGBAMaterial()
+        : Material(uniforms()) {
+}
+
+// static
+const AttributeSet& RGBAMaterial::attributes() {
+    static AttributeSet set = makeAttributeSet<QVector2D, QVector4D>({"position", "color"});
+    return set;
+}
+
+/* static */ const UniformSet& RGBAMaterial::uniforms() {
```

Comment:
super nit -  just for consistency with the above
```suggestion
// static
const UniformSet& RGBAMaterial::uniforms() {
```